tests/automation_framework/data/worker/workerregister_proofData_empty.json

```diff
@@ -0,0 +1,26 @@
+{
+    "jsonrpc": "2.0",
+    "method": "WorkerRegister",
+    "id": 1,
+    "params": {
+        "workerId":"",
+        "workerType": 1,
+        "organizationId":"0Xd",
+        "applicationTypeId": "0XAD",
+        "workerEncryptionKey" : " " ,
+        "details": {
+            "hashingAlgorithm": "SHA-256",
+            "signingAlgorithm": "SECP256K1" ,
+            "keyEncryptionAlgorithm": "RSA-OAEP-3072" ,
+            "dataEncryptionAlgorithm": "AES-GCM-256" ,
+            "workerTypeData": {
+                "verificationKey": "0xA",
+                "proofDataType": null,
```

Comment:
test name says proof data empty. But we are passing null string. Is this right??

Reply:
null also means empty.

Reply:
Anjali, as per the current convention when we say we mean empty string like "", and when we have to pass null we specifically call null. This is because most of the params take string input and we have tests where we wanted to pas empty string, so for this one can you rename the test to prooddata_null?

Reply:
So, shall i create an additional test with string as empty

Reply:
Go ahead, work with Manoj to add the test in QC.